app/utils/domElements.js

```
@@ -0,0 +1,91 @@
+export class DOMElements {
+  static instance = null;
+
+  constructor() {
+    if (DOMElements.instance) {
+      return DOMElements.instance;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)